### ISSUE TITLE
DOMA with fixed values - clear DOMNAME

### ISSUE THREAD

**Issue by FreHu** [abapGit/abapGit#4937]
All the fixed values contain the domain name again, this seems redundant
![image](https://user-images.githubusercontent.com/5097067/132747905-a575e104-cf41-4294-91c3-dda4834aa0ce.png)

**Comment by larshp** [comment]
agree, clear at serialization, add again at deserialization, just to make sure not to break anything

**Comment by FreHu** [comment]
This is already done during deserialize:
https://github.com/abapGit/abapGit/blame/main/src/objects/zcl_abapgit_object_doma.clas.abap#L266-L269

Per your comment it should be okay to clear both `domname` and `valpos` now
https://github.com/abapGit/abapGit/pull/2981#issuecomment-554729559

**Comment by larshp** [comment]
yea

not sure if this is something to be done now, or wait annoying the users until moving to json files

**Comment by FreHu** [comment]
if it was started it makes sense to finish it, currently it's filled back during deserialize for no reason

**Comment by larshp** [comment]
yea, json is also somewhat out in the future
